app/__init__.py

```diff
@@ -328,6 +330,22 @@ def make_session_permanent():
     session.permanent = True
 
 
+def create_url(environment, url):
+    url_created = urlparse(url)
+    if environment == "production":
+        url_list = list(url_created)
+        url_list[1] = "beta.notify.gov"
+        url_for_redirect = urlunparse(url_list)
+        return url_for_redirect
+    return url
```

Comment:
This is a great start!  There's a couple more things we could probably do here.

One, as noted below, I don't think we need to check for the environment in this method - it can happen outside of the method, which means this becomes more focused on the task itself (it'd likely also help with code coverage just a bit too; one less branching code path with the removal of the `if` statement in here.

Two, we may need to do a bit more with the swapping of the `netloc` piece of the URL to make sure we're replacing it as we intend to and only when we intend to, but I'm not 100% sure just yet.

With that said, one thing at a time:  let's simplify this a bit and make it strictly about creating the beta URL.

```suggestion
def create_beta_url(url):
    url_created = urlparse(url)
    url_list = list(url_created)
    url_list[1] = "beta.notify.gov"
    url_for_redirect = urlunparse(url_list)

    return url_for_redirect
```